server/services/caseloadService.ts

```diff
@@ -79,10 +79,16 @@ export default class CaseloadService {
         //  fix will be to have an endpoint to search nomis by CRD (CVSL-492).
         //  Temporary fix is to filter only prisoners who have a CRD in the next 13 weeks before searching them on delius
         return caseload
-          .filter(offender => offender.conditionalReleaseDate)
-          .filter(offender => moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isSameOrAfter(moment(), 'day'))
-          .filter(offender =>
-            moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isBefore(moment().add(13, 'weeks'), 'day')
+          .filter(offender => offender.conditionalReleaseDate || offender.confirmedReleaseDate)
+          .filter(
+            offender =>
+              offender.conditionalReleaseDate &&
+              moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isSameOrAfter(moment(), 'day')
+          )
+          .filter(
+            offender =>
+              offender.conditionalReleaseDate &&
+              moment(offender.conditionalReleaseDate, 'YYYY-MM-DD').isBefore(moment().add(13, 'weeks'), 'day')
           )
```

Comment:
I know this is a temporary block... but people will still be filtered from OMU view if they dont have a CRD by these rules

Reply:
Maybe set a variable called date which == `offender.confirmedlReleaseDate || offender.conditionalReleaseDate` before doing the date checks

Reply:
I saw this bit and did wonder - it duplicate the rules, but its to cut-down the numbers going into the next  functions temporarily.  Probably does need opening up in a similar way to the eligibility filter i.e. conditionalReleaseDate OR confirmedReleaseDate.

Reply:
On it now.